compiler/ballerina-parser/src/test/resources/declarations/class-def/class_def_source_35.bal

```diff
@@ -1,5 +1,6 @@
-class client client client Foo {
-    string name;
+class 
+client client client Foo {
+string name;
```

Comment:
what is the need for this change?

Reply:
Recovered as,
```
class MISSING[] MISSING[{] 
INVALID[client]  INVALID[client]  INVALID[client] Foo  INVALID[{]
 INVALID[string] name;
}
```
and there for is formatted as above.

Reply:
Okay